charts/internal/seed-controlplane/charts/csi-driver-controller/values.yaml

```diff
@@ -12,46 +12,74 @@ images:
   csi-snapshot-validation-webhook: image-repository:image-tag
 
 socketPath: /var/lib/csi/sockets/pluginproxy
-timeout: 3m
 userAgentHeaders: []
 
-resources:
-  driver:
+csiDriverCinder:
+  resources:
     requests:
       cpu: 20m
       memory: 50Mi
     limits:
-      memory: 360Mi
-  provisioner:
+      cpu: 50m
```

Comment:
Please adopt here the old memory limits and drop the cpu limits.
We recently changed them to more accurate values. The values here would revert that.